registry/insecure.md

```diff
@@ -77,7 +77,7 @@ This is more secure than the insecure registry solution.
     depends on your OS.
 
     - **Linux**: Copy the `domain.crt` file to
-      `/etc/docker/certs.d/myregistrydomain.com:5000/ca.crt` on every Docker
+      `/etc/docker/certs.d/myregistrydomain.com/ca.crt` on every Docker
```

Comment:
@dmcgowan could you have a look? IIRC the `:port` was needed in the path (as different registries could run on different ports), but not sure

Reply:
The port is needed if the registry is not on 80 or 443